server/chat-plugins/scavengers.js

```diff
@@ -440,15 +441,15 @@ class ScavengerHunt extends Rooms.RoomGame {
 		}
 	}
 
-	onSendQuestion(user) {
+	onSendQuestion(user, showHints) {
 		if (!(user.userid in this.playerTable) || this.hosts.some(h => h.userid === user.userid)) return false;
 
 		let player = this.playerTable[user.userid];
 		if (player.completed) return false;
 
 		let current = player.getCurrentQuestion();
 
-		player.sendRoom(`|raw|You are on ${(current.number === this.questions.length ? "final " : "")}hint #${current.number}: ${Chat.formatText(current.question.hint)}`);
+		player.sendRoom(`|raw|You are on ${(current.number === this.questions.length ? "final " : "")}hint #${current.number}: ${Chat.formatText(current.question.hint)}${showHints && current.question.spoilers ? `<details><summary>Extra Hints:</summary>${current.question.spoilers.map(p => `- ${p}`).join('<br />')}</details>` : ''}`);
```

Comment:
I think this should check whether the hunt has hints, otherwise you end up with confusing `<details>` that do nothing when clicked.